VersionControlUI/src/DiffManager.h

```diff
@@ -141,12 +149,20 @@ class VERSIONCONTROLUI_API DiffManager
 
 		bool processNameChange(Model::Node* oldNode, Model::Node* newNode, const DiffSetup& diffSetup);
 
-		void showNameChangeInformation(Visualization::ViewItem* currentViewItem, const DiffSetup& diffSetup);
+		void showNameChangeInformation(Visualization::ViewItem* currentViewItem, const DiffSetup& diffSetup,
+												 VersionControlUI::DiffComparisonPair* diffComparisonPair);
+
+		bool satisfiesTargetNodeIdConstraint(Model::Node* oldNode, Model::Node* newNode,
+																					const DiffSetup& diffSetup);
+
+		bool satisfiesNameChangeVisualizationConstraints(Model::NodeIdType id);
 
 		QString project_;
 		QList<Model::SymbolMatcher> contextUnitMatcherPriorityList_;
-		Model::NodeIdType targetNodeID_;
+		Model::NodeIdType targetNodeId_;
```

Comment:
It is used to specify which node id we are interested in. At the moment it is only used for history. OK, will do.